### PR TITLE
add 7.0/fpm dockerfiles

### PR DESCRIPTION
**Pull request by srspnda**

### CODE REVIEW

File: 7.0/fpm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	&& curl -sSL https://getcomposer.org/installer \
+		| "$PHP_PREFIX_DIR/bin/php" \
+		-- --install-dir="$PHP_PREFIX_DIR/bin" \
+	&& useradd php \
+	&& { find /usr/local/bin /usr/local/sbin -type f -executable -exec strip --strip-all '{}' + || true; } \
+	&& apt-get purge -y --auto-remove \
+		-o APT::AutoRemove::RecommendsImportant=false \
+		-o APT::AutoRemove::SuggestsImportant=false $PHP_BUILD_DEPS \
+	&& make clean \
+	&& rm -rf /src/php
```
Comment (sodabrew) [inline 7.0/fpm/Dockerfile]: Great to have the cleanup on these packages, so they're not in the output Docker image

**Comment by sodabrew** [comment]
LGTM. Will this automatically appear in https://hub.docker.com/r/life360/php after merge?

**Comment by srspnda** [comment]
I'll have to setup the build; then all subsequent commits to master within /7.0/fpm/ directory will build the tags i setup

**Comment by srspnda** [comment]
There, now it will.

**Pull request merged**